src/stylesheets/components/_card.scss

```diff
@@ -191,8 +191,12 @@ $inner-border-radius: calc(#{$sds-card-border-radius} - #{$sds-card-border-width
   &--vertical {
 
     @include at-media('mobile') {
+      @include u-display('flex');
+
       .sds-card__header {
-        text-align: center;
+        align-items: center;
+        justify-content: left;
+        text-align: left; 
```

Comment:
We should retain the border right that is currently only at the tablet level. Move that to mobile and larger size.

Reply:
got it.